core/src/components/alert/alert.scss

```
@@ -197,3 +197,7 @@
 .alert-checkbox-inner {
   box-sizing: border-box;
 }
+
+textarea.alert-input {
+  min-height: $alert-input-min-height;
```

We need to disable horizontal resizing since you can resize the textarea off screen.

```suggestion
  min-height: $alert-input-min-height;
  resize: vertical;
```

---

I thinks I did that before !!
maybe it comes back by mistake while merging with master let me check it again